lib/cc-reno.c

```diff
@@ -55,19 +58,28 @@ static void reno_on_acked(quicly_cc_t *cc, const quicly_loss_t *loss, uint32_t b
 void quicly_cc_reno_on_lost(quicly_cc_t *cc, const quicly_loss_t *loss, uint32_t bytes, uint64_t lost_pn, uint64_t next_pn,
                             int64_t now, uint32_t max_udp_payload_size)
 {
+    /* when exiting slow start, use inverse of exponential growth ratio, as loss is detected 1 RTT later, at which point CWND has
+     * overshot as much as the growth ratio */
+    double beta = cc->ssthresh == UINT32_MAX ? 0.5 : QUICLY_RENO_BETA;
```

Comment:
It is a bit questionable if we want to make this change.

While it is true that 0.5 gives us the correct bandwidth estimate, it does reduce the aggressiveness of the CC.

With the original beta of 0.7, we have been essentially doing a steep congestion avoidance phase right after slow start. Once that congestion avoidance phase concludes, we end in the optimal state (0.7 * 0.7 ~~ 0.5).

Reply:
Now, all the CCs (i.e., incl. Cubic) do 0.5x when observing the first loss. Doing so makes sense unless HyStart++ is used, but we do not have the code yet.